pom.xml

```diff
@@ -127,7 +127,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <java.version>1.6</java.version>
     <hadoop.version>1.0.4</hadoop.version>
-    <glusterfs-hadoop.version>2.3.5</glusterfs-hadoop.version>
+    <glusterfs-hadoop.version>2.3.13</glusterfs-hadoop.version>
```

Comment:
yes, that is necessary. 2.3.5 constructs scheme from path, if the path is not well formed, it may get null pointer.

2.3.13 will return a default scheme 
https://github.com/gluster/glusterfs-hadoop/blob/master/src/main/java/org/apache/hadoop/fs/glusterfs/GlusterVolume.java#L73